scan/rule.go

```diff
@@ -121,6 +121,10 @@ func (repo *Repo) CheckRules(bundle *Bundle) {
 						continue
 					}
 
+					if 0 < rule.ReportGroup && rule.ReportGroup < len(groups) {
```

Comment:
**question:** Would it be worth adding a comment clarifying `0` is a submatch for the entire expression rather than a zero-count of capture groups? I know this is fairly universal so maybe I'm just not in a super regexy mindset but it was a thought since it took me a little bit to parse this one. Feel free to disregard

Reply:
I added a comment. What do you think?

Reply:
This is clear to me 👍